lib/process-base.js

```diff
@@ -0,0 +1,29 @@
+module.exports = class ProcessBase {
```

Comment:
```javascript
class Bla {
...
}

module.exports = Bla;
```

Reply:
When people go through our code to search for what `require('./process-base')` returns they will go to this file and first search for `module.exports`, only to see that the `ProcessBase` class is exported and then search for it.

I'm saving the second Ctrl+F. What are the advantages to your way?

Reply:
Again
Flip a coin 
We talked about it for like thousand times before